### ISSUE TITLE
Upgrade all profiles ability

### ISSUE THREAD

**Issue by BlackBaroness** [gorilla-devs/ferium#270]
## The problem
Currently we can't just update all mods inside all profiles that can be important for someone (it is for me).

## My solutions
Add a command such a `ferium upgrade --all-profiles` to run upgrade for all profiles.

**Comment by theRookieCoder** [comment]
The PR got me thinking, how should this be implemented? Should it upgrade each profile separately or should all the profiles be upgraded at once? (I prefer the latter)

**Comment by BlackBaroness** [comment]
> The PR got me thinking, how should this be implemented? Should it upgrade each profile separately or should all the profiles be upgraded at once? (I prefer the latter)


I don't know exactly how the architecture of this application is structured. But I imagine something like this:
```
for (profile : profiles) {
    profile.upgrade()
}
```

I think it's probably best to just run upgrade for each profile separately. I tried to do this with a bash script, but the parsing turned out to be too complicated.

(Just looked at PR and it's exactly as I imagined)

Updating all profiles at the same time will require too much code change, it seems to me that this option would still be more correct.
